namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java

```diff
@@ -635,44 +635,35 @@ private RemotingCommand updateConfig(ChannelHandlerContext ctx, RemotingCommand
         return response;
     }
 
-    private RemotingCommand getConfig(ChannelHandlerContext ctx, RemotingCommand request) {
+   private RemotingCommand getConfig(ChannelHandlerContext ctx, RemotingCommand request) {
         final RemotingCommand response = RemotingCommand.createResponseCommand(null);
 
         String content = this.namesrvController.getConfiguration().getAllConfigsFormatString();
+        setResponse(response, content);
+        return response;
+    }
+
+    private void setResponse(RemotingCommand response, String content) {
```

Comment:
Occurrence of the extracted method is only 1?

Reply:
no,two method,getClientConfigs(ChannelHandlerContext ctx, RemotingCommand request)  and getConfig(ChannelHandlerContext ctx, RemotingCommand request).